src/akkudoktoreos/class_ems.py

```diff
@@ -54,12 +54,11 @@ def simuliere_ab_jetzt(self) -> dict:
         return self.simuliere(start_stunde)
 
     def simuliere(self, start_stunde: int) -> dict:
-        """
-        hour:
-            akku_soc_pro_stunde begin of the hour, initial hour state!
-            last_wh_pro_stunde integral of  last hour (end state)
-        """
+        """hour.
 
+        akku_soc_pro_stunde begin of the hour, initial hour state!
+        last_wh_pro_stunde integral of  last hour (end state)
+        """
```

Comment:
```suggestion
        """Simulate energy usage for given profiles.
        
        Args:
            start_stunde
        """
```

Reply:
Hello @michaelosthege,

I only did changes to the format of the comments to pass the ruff rules for comments. Content change should be part of another PR by the author or somebody who knows the topic.